app/export.go

```diff
@@ -37,14 +37,17 @@ func (app *RegenApp) ExportAppStateAndValidators(
 		genState[name] = v
 	}
 
-	// Export genesis state from new modules (that use ADR 33 approach)
-	// if they are not already part of genState
-	for name, v := range app.smm.ExportGenesis(ctx) {
-		if _, ok := genState[name]; ok {
-			return servertypes.ExportedApp{}, fmt.Errorf("genesis state already exported for %s module", name)
-		}
-		genState[name] = v
-	}
+	// TODO(Tyler): do we need below??
+
+	//// Export genesis state from new modules (that use ADR 33 approach)
+	//// if they are not already part of genState
+	//for name, v := range app.mm.ExportGenesis(ctx)
+	//for name, v := range app.smm.ExportGenesis(ctx) {
+	//	if _, ok := genState[name]; ok {
+	//		return servertypes.ExportedApp{}, fmt.Errorf("genesis state already exported for %s module", name)
+	//	}
+	//	genState[name] = v
+	//}
```

Comment:
I don't believe so. Genesis exports should be handled by the sdk module manager.